PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Set ReleaseChannel In JumpCloud Password Manager.md

```diff
@@ -0,0 +1,35 @@
+#### Name
+
+Windows - Set JumpCloud Password Manager's Release Channel | v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+# Set $RELEASE_CHANNEL to beta OR dogfood OR public depending on your desired release channel
```

Comment:
We are not noting the line here, and I think that's the correct way to tell customers what to do, since it's really easy to forget that we need to update that in the future.